packages/apputils-extension/src/index.ts

```diff
@@ -298,6 +303,51 @@ const splash: JupyterFrontEndPlugin<ISplashScreen> = {
   }
 };
 
+const print: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/apputils-extension:print',
+  autoStart: true,
+  requires: [ILabShell],
+  activate: (app: JupyterFrontEnd, labShell: ILabShell) => {
+    // TODO: This was copied out of application-extension/src/index.tsx
+    // This should be refactored to not duplicate the code.
+    // Returns the widget associated with the most recent contextmenu event.
+    const contextMenuWidget = (): Widget => {
```

Comment:
This `contextMenuWidget` function is pretty much *only* needed to dispatch the command to the right widget when the user right-clicks on one of the main area tabs. If we are not adding the command to the tab-area-context-menu, then I think we can cut this function entirely.

Reply:
Sounds good!